package.json

```diff
@@ -1,18 +1,26 @@
 {
   "name": "lighthouses",
-  "version": "0.0.1",
+  "version": "2.0.0",
   "description": "",
-  "main": "index.js",
+  "main": "index.ts",
   "type": "module",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "type-check": "tsc --noEmit"
   },
-  "author": "",
+  "author": "@charkour, @TheDunco",
```

Comment:
I don't believe this is a valid value for `"author"`.

You can omit this field and replace it with `"collaborators"` which takes an [array of people](https://docs.npmjs.com/cli/v10/configuring-npm/package-json#people-fields-author-contributors). Thanks!

Reply:
Roger, will do! I'm not really familiar with `package.json` standards and practices and options for those types of things, so happy to learn more and do this the correct way!

Reply:
All good! There's a lot to learn. I've battled with `package.json` configuration a bunch with my open-source work. Happy to help review your changes.